packages/radix-ui-themes/src/styles/tokens/radius.css

```diff
@@ -4,7 +4,7 @@
   --radius-full: 0px;
 
   &:where([data-radius]),
-  & [data-radius] {
+  & :where([data-radius]) {
```

Comment:
Any reason we wouldn't define radius tokens targeting just `[data-radius]` rather than `.radix-themes[data-radius] and `.radix-themes [data-radius]`? The result would be effectively the same and we use simpler selectors for color accents.

Reply:
All tokens are defined inside `.radix-themes` so there's no leakage. I see what you're saying, but the only thing that makes it a bit icky for me is the fact that the attribute is pretty generic…

Reply:
Updated to cleaner selectors where possible as discussed (https://github.com/radix-ui/themes/pull/79/commits/2537fa943119104cb1c41edd5db9ff831f721249)